### ISSUE TITLE
Readme and ModelAdmin

### ISSUE THREAD

**Issue by andybak** [django-guardian/django-guardian#419]
In the readme it says:

> Replace admin.ModelAdmin with GuardedModelAdmin

Trouble is - every app says the same thing and it's always a worry whether you can integrate two different apps that both want to 'own' ModelAdmin.

Ideally the advice should be: 

> Make sure your ModelAdmin's use GuardedModelAdmin as a mixin

with some advice on potential conflicts. Do all your overrode methods call super() and will they behave nicely in an inheritance chain?

I had a quick look at GuardedModelAdmin and it overrides templates. This is obviously problematic as the last mixin wins the template property. Is there any documentation for what the custom templates do and alternative ways to achieve the same thing (i.e. a templatetag the end user can drop into their own custom templates?

**Comment by brianmay** [comment]
I haven't looked much at this Admin stuff yet, so can't really comment much. Plus at the moment I don't have any time to look. Any suggestions or PR to improve the code appreciated :-)

I suspect we might override the templates so we can add extra elements, such as buttons, required for django-guardian. However I haven't double checked this. Nor can I think of any way to make this cleaner for your use case.

**Comment by andybak** [comment]
OK. It's a problem I've been meaning to tackle in the general case for a while now. I'm not currently using Django Guardian but it's not a problem that is specific to Django Guardian. When I've found a good approach for other apps I'll come back with some specific suggestions and hopefully a PR.

**Comment by thclark** [comment]
I agree this would be more convenient as a mixin to avoid that collision with other third party apps. That also facilitates further customization as you can subclass your own mixin variant and place it wherever you like.

Happy to accept a PR for this.

**Comment by Natgho** [comment]
https://django-guardian.readthedocs.io/en/stable/api/admin/#guardian.admin.GuardedModelAdminMixin

it is already exist?

**Comment by thclark** [comment]
Oh yeah my baad, great, one less thing to do. Well spotted @Natgho
